week1/assignment/todo-loi/todo.rb

```diff
@@ -0,0 +1,123 @@
+require_relative "item"
+require_relative "list"
+require 'colorize'
+
+class Todo
+  attr_accessor :list
+  def initialize(file_name = "todo.data")
+    puts ("-" * 75).colorize(:green)
+    puts "Welcome to the Ruby Todo List App!".colorize(:green)
+    puts "If you'd like to see a list of available comments, type in \n" + "-h".colorize(:light_red) +  " or " + "help".colorize(:light_red) +  " to " + "view the help menu.\n\n".colorize(:light_red)
+    file = File.open(file_name)
+    data = file.read
+    items = []
+    data.lines.each do |line|
```

Comment:
instead of `.lines`, can use `split.("\n")` to avoid having to chomp inside `Item#initialize`